packages/runtime/src/templates/getHandler.ts

```diff
@@ -160,32 +169,49 @@ const makeHandler = ({ conf, app, pageRoot, NextServer, staticManifest = [], mod
     }
 
     // Sending SWR headers causes undefined behaviour with the Netlify CDN
```

Comment:
Worth deleting this comment as it probably doesn't hold true now?

Reply:
Probably worth updating or maybe revisiting. As it's not just CDN behavior that it would affect - it would also affect browser cache behavior - if we had both SWR on CDN and in browser, it would mean that content that CDN sends back that might already be stale (while it starts to revalidate it in background) would be fresh for browser and browser would potentially cache response it sees as fresh but might actually already be stale (in CDN terms)